main.cs

```diff
@@ -1008,6 +1010,7 @@ public enum AdditionalWinners
     Phantom = CustomRoles.Phantom,
     Maverick = CustomRoles.Maverick,
     Shaman = CustomRoles.Shaman,
+    Taskinator = CustomRoles.Taskinator,
```

Comment:
You forgot add `AdditionalWinners.Taskinator` in `en_US.json`

Reply:
done